portfolio/src/main/webapp/script.js

```diff
@@ -11,3 +11,25 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+/** Fetches tasks from the server and adds them to the DOM. */
+function loadComments() {
+  fetch('/input').then(response => response.json()).then((comments) => {
+    const commentsListElement = document.getElementById('comments-list');
+    comments.forEach((comment) => {
+      commentsListElement.appendChild(createTaskElement(comment));
+    })
+  });
+}
+
+/** Creates an element that represents a task, including its delete button. */
+function createTaskElement(comment) {
+  const commentElement = document.createElement('li');
+  commentElement.className = 'comments';
+
+  const titleElement = document.createElement('span');
+  titleElement.innerText = comment.entry;
+
+  commentElement.appendChild(titleElement);
+  return commentElement;
+}
```

Comment:
Thanks for adding a method to do this. Makes much cleaner and easier to understand the code.

Reply:
Thank you! :)